tests/plans.offline.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { expect, test } from "@playwright/test";
+
+import { selectFacultyAndRegistration } from "./helpers";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/plans");
+});
+
+test.describe("Plans logic offline", () => {
+  test("should display plans page", async ({ page }) => {
+    await expect(page).toHaveTitle(/Planer - utwórz swój plan zajęć na PWR!/i);
+    await expect(
+      page.getByRole("heading", { name: "Planer", level: 1 }),
+    ).toBeVisible();
+  });
+
+  test("should display login button", async ({ page }) => {
+    await expect(page.getByText(/Zaloguj się/i)).toBeVisible();
+  });
+
+  test("should create empty plan", async ({ page }) => {
+    const button = page.getByTestId("create-plan-button");
```

Comment:
szkoda, że użyłeś testId - powinno się móc do tego dostać w jakiś prostszy sposób. jak nie - to znaczy, że a11y jest słabe